sites/public/lib/HorizontalScrollSection.tsx

```diff
@@ -12,27 +12,37 @@ export interface HorizontalScrollSectionProps {
   icon?: IconTypes
 }
 
+function isRtl(): boolean {
```

Comment:
It seems like the component isn't getting updated when the language changes. I think you might need to use a [MutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver) to watch for when document.body.direction changes? My guess is you probably want to add that to useEffect and clean it up as well

Reply:
Good catch, I was only manually testing when the view was already scrolled a little bit, so both buttons were already enabled.